data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/illuminates/illuminates_no_max_value.cfg

```diff
@@ -0,0 +1,31 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [illuminates]max_value=
+##
+# Actions:
+# Give all units 50% self-illuminates bonus damage
+# Attack each other
+##
+# Expected end state:
+# The damage from the attack is not increased
```

Comment:
I've very confused - why doesn't the bonus take effect in this case?

Reply:
It has the same issue as [resistance] previously did where max_value defaults to 0 if it's not present.

Reply:
Ah, understood. Please would you add a comment clarifying that the test is checking the default value?

Reply:
Added.